reference/log/log.txt

```diff
@@ -0,0 +1,5 @@
+	v0: REQUIRED level of info
```

Comment:
Mmmmm... I don't think logs from our runs (I assume that's what this is?) should be included in the main code data base. I think it's fine to put the logs in this location, I would just have the logs be ignored. (so make an update to the gitignore file?)